app/views/editor/questionRich.html.erb

```diff
@@ -0,0 +1,313 @@
+      <h4><i class="fa fa-question-circle"></i> Use this form to ask Question about <b><%= @node.power_tag('question') if @node %></b></h4>
+      <% if @node && DrupalTag.follower_count(@node.power_tag('question')) > 0 %>
+        <div class="alert alert-success" role="alert"><%= DrupalTag.follower_count(@node.power_tag('question')) %> people have signed up to answer questions on this topic.</div>
+      <% end %>
+      <p>Here you can ask a question to the Public Lab community to help you in your environmental exploration. This is a great space to ask whether a method exists for identifying a possible pollutant, or ask about details of a specific technique for identifying a pollutant.</p>
+      <p>You can add more tags in the tags section below to further relate your question. You can also change the main tag of the question by renaming the part after <b><i>question:</i></b> tag</p>
+      <p>New to Public Lab? Take a minute to read the Q &amp; A wiki before posting your question</p>
+      <h4><a href="//<%= request.host %>/wiki/public-lab-q-and-a">Learn more &raquo;</a></h4>
+    </div>
+
+  <% else %>
+    <h3 class="visible-md"><i class="fa fa-file"></i> What's a <br />research note?</h3>
+    <h4 class="hidden-md"><i class="fa fa-file"></i> What's a research note? <a onClick="$('#research-note-detail').toggleClass('visible-sm')">Learn more &raquo;</a><br /></h4>
+
+    <div id="research-note-detail" class="hidden-sm hidden-xs">
+      <p>At Public Lab, we all collaborate to invent and improve open source environmental science tools. <b>Research notes</b> are the primary way we share what we learn, ask questions, pose challenges, and critique one another's work.</p>
+      <p><b>Research notes can include:</b></p>
+      <ul>
+        <li>photos of what you've made</li>
+        <li>requests for troubleshooting</li>
+        <li>proposals for new projects</li>
+        <li>announcements of events</li>
+        <li>reports from a field test or meetup</li>
+        <li>etc!</li>
+      </ul>
+      <hr />
+      <p><b><a href="//www.youtube.com/watch?v=rsy__zTTh8c" target="_blank"><i class="fa fa-film"></i> Watch a 30 second video on posting research notes &raquo;</a></b></p>
+      <hr />
+      <div class="sidebar-nav">
+        <ul class="nav nav-list">
+          <li class="nav-header">More</li>
+          <li><a href="/getting-started">Getting started with Public Lab</a></li>
+          <li><a href="/wiki/posting-research">About posting research</a></li>
+          <li><a href="/wiki/authoring-help#Advanced+formatting">Advanced formatting</a></li>
+        </ul>
+      </div>
+    </div>
+  <% end %>
+</div>
+
+<div class="col-sm-9">
+  <%
+  if @node && params[:action] != "create"
+    url = { :controller => "notes", :action => "update", :id => params[:id] }
+  else
+    url = { :controller => "notes", :action => "create", :id => params[:id] }
+  end
+  %>
+  <%= form_for @node, :as => :drupal_node, :url => url do |f| %>
```

Comment:
Here's where the second form is generated. But actually upon reading through it, I don't think there's anything in particular you need to save from this older form. I think you can just delete it and get this template (and its corresponding controller) to match `rich.html.erb` as closely as you can. 

Then we can take the next steps on the design and getting it to look like the mockup.